### PR TITLE
Added Base-N Calculator files

### LINKED ISSUE #1044: A script that converts a number of base-x to base-y
# Description

A more friendly title `` Base-N Calculator ``. A script that gets 3 inputs from the user:
* Number
* Number base (base-x)
* Base of the converted number (base-y)

## Language

- [x] Python

## Checklist:

* The script will not only be limited to known bases `` Decimal \ Octal \ Binary \ Hexadecimal ``.
* User will be able to convert to any base as long as the base value is not smaller than 2.
* The script will be able to deal with decimals numbers.
* No external libraries will be used for conversion. (Imparative approach)

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Base-N Calculator | Converts a number of base-x to base-y | GSSOC `` label: GSSOC21 `` | XZANATOL |

### PR DESCRIPTION
**Pull request by XZANATOL**
# Description

Added Base-N calculator files.

Fixes #1044 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by XZANATOL** [comment]
@vybhav72954 Hi there, :D
I'm going to DM you this night (It's 6:00 AM in my timezone rn) about something considering the recently added GitHub Action. Can you take some time in reviewing this one for the mean time?

File: Python/Base-N-Calc/Readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Enter Base-y > 10
+
+Result: 87.0
+```
+
+# Authors
+
+Written by [XZANATOL](https://www.github.com/XZANATOL).
+
+The project was built as a contribution during [GSSOC'21](https://gssoc.girlscript.tech/).
```
Comment (vybhav72954) [inline Python/Base-N-Calc/Readme.md:31]: Remove l31 please

File: Python/Base-N-Calc/script.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        # Add element to result
+        res += element
+        i += -1
+
+    # Return total result
+    return res
+
+
+def Con_Dec_to_Base_Y(num, base_y):
+    # Converting the integeral part
```
Comment (vybhav72954) [inline Python/Base-N-Calc/script.py:33]: We need docstrings not comments

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
Hey @XZANATOL Sorry for delay, everything looks good, just 1 major change: Missing docstrings for every function

**Comment by sohamsshah** [comment]
Are you working on this @XZANATOL ?

**Pull request closed without merging**